packages/mock-block-dock/dev/test-react-block.tsx

```diff
@@ -2,6 +2,7 @@ import {
   BlockComponent,
   useGraphBlockService,
 } from "@blockprotocol/graph/react";
+import { useHook, useHookBlockService } from "@blockprotocol/hook/react";
```

Comment:
Added use of the hook service to the React test block - MBD dev environment only